README.md

```diff
@@ -1 +1,168 @@
+    </ul>
+
+### Key Features <a name="key-features"></a>
+
+- A login page.
+- A registration page.
+- A food list (with all CRUD implementation, except for 'update').
+- A recipes list (with all CRUD implementation, except for 'update').
+- A general shopping list view (all missing food for all your recipes and total price).
+- A list of all public recipes from other users with their names and total prices.
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!---pending->
+
+<!-- LIVE DEMO -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+  - A browser of you choice.
+  - A text editor of your choice.
+  - Install ruby gem and depencies on your local system
+  - Install Postgresql
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Use the following Commands:
+      https://github.com/Tshobohwa/recipe-app
+      cd recipe-app   
+
+
+### Usage
+-bundle install
+- rails s
+
+
+### Run tests
+- Run the following script and style test:
+<!---To be done-->
+      
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Frank Mugagga**
+
+- GitHub: [@githubhandle](https://www.github.com/FrankMugagga)
+- Twitter: [@twitterhandle](https://www.twitter.com/@mugagga_frank)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/frank-mugagga)
+
+👤 **Salomon Tshobobwa**
+
+- GitHub: [@githubhandle](https://github.com/Tshobohwa)
+- Twitter: [@twitterhandle](https://twitter.com/SalomonTshoboh1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tshobohwasalomon/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Deploy]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Tshobohwa/recipe-app/issues)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+ 
+  If you like this project, give it a star and share it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+  I would like to thank Microverse and coding partner.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **How to make it mobile friendly?**
+
+  - Put a viewport tag in the header
+
+- **How to design the site?**
+
+  - Draw a mockup before start to code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Tshobohwa/recipe-app/tree/dev) licensed.
```

Comment:
- [ ] Kindly fix your Licence link to make your readme file looks professional 🙏

Reply:
Thank you for your review @ichala 😄